pages/science/flames.rst

```diff
@@ -83,8 +83,9 @@ heat capacity of species :math:`k`, :math:`h_k` is the enthalpy of species
 :math:`k`, :math:`W_k` is the molecular weight of species :math:`k`, and
 :math:`\dot{\omega}_k` is the molar production rate of species :math:`k`.
 
-The tangential velocity :math:`w` has been assumed to be zero, and the fluid has
-been assumed to behave as an ideal gas.
+The tangential velocity :math:`w` has been assumed to be zero. The model is
+applicable to both ideal and non-ideal fluids, which follow ideal-gas or real-gas 
+(Redlich-Kwong and Peng-Robinson) equations of state.
```

Comment:
Can you mention that the real gas support is new in Cantera 3.0?

Reply:
I have added a sentence mentioning that.